src/lamplib/src/genny/tasks/genny_runner.py

```diff
@@ -34,6 +33,12 @@ def main_genny_runner(
         workspace_root=workspace_root,
         genny_repo_root=genny_repo_root,
     ):
+        # Inserting an Info log to gather some information 
+        # about what version of Python Genny is running
+        # This will help inform whether upgrading packages that deprecate
+        # use of Python versions that Genny supports will cause problems
+        SLOG.info(f"Genny is running the Python version: {sys.version}")
```

Comment:
How about we make this log line more structured? (I added the quotes around the output of `sys.version` to (hopefully) make it easier to parse, as the output is a string with multiple spaces)

```suggestion
        SLOG.info(f"Python version being used by Genny", python_version=f'"{sys.version}"')
```

Reply:
Sounds great, thank you!